src/form/Form.tsx

```diff
@@ -1191,16 +1157,31 @@ function Form({
     }));
   };
 
-  const buttonOnSubmit = async (button: any, submitData: any) => {
+  const buttonOnSubmit = async (
+    button: any,
+    submitData: boolean,
+    buttonAction?: any
+  ) => {
     try {
       const metadata = {
         elementType: 'button',
         elementIDs: [button.id]
       };
       if (submitData) {
+        // ************************************************
+        // ************************************************
+        // ************************************************
+        // ************************************************
+        // what about the case where we are going to nav, but not submit?
```

Comment:
if the button is configured like this
<img width="354" alt="image" src="https://user-images.githubusercontent.com/923485/201222645-f14867d8-7ebe-4019-a5e9-b5e27a67a215.png">

should we execute buttonAction? This code does *not* execute the btn action in this case. I think it probably should, which is overall an easy change

in this case, since validation is turned off - it should just attempt to execute buttonAction without setting any errors because it the expected bx is to transition to the next step no matter what?